### PR TITLE
Add support to get raw file content as byte[]

### LINKED ISSUE #1651: The client doesn't support for files raw content
I didn't found any object that refers to all files of the repository. Could you please tell me if it is correct ? 

Thanks in advance!

Lucas

### PR DESCRIPTION
**Pull request by 0xced**
This allows to spare a little over 100ms in my tests on small files vs using the `GetAllContentsByRef` API.

Fixes #1651

### CODE REVIEW

**Comment by 0xced** [comment]
@shiftkey I just rebased onto master. Do you think you'll get some time to review this pull request sooner or later?

File: Octokit/Helpers/ApiUrls.cs [hunk 1]
```diff
@@ -2363,7 +2363,7 @@ public static Uri RepositoryArchiveLink(string owner, string name, ArchiveFormat
         /// <returns>The <see cref="Uri"/> for getting the contents of the specified repository and path</returns>
         public static Uri RepositoryContent(string owner, string name, string path, string reference)
         {
-            return "repos/{0}/{1}/contents/{2}?ref={3}".FormatUri(owner, name, path, reference);
+            return "repos/{0}/{1}/contents/{2}?ref={3}".FormatUri(owner, name, path == "/" ? "" : path, reference);
```
Comment (shiftkey) [inline Octokit/Helpers/ApiUrls.cs:2366]: This is a nice bit of cleanup for my more explicit example above - thanks!

**Comment by shiftkey** [comment]
release_notes: added `GetRawContent` and `GetRawContentByRef` APIs to Repository Contents client for cases where performance is desirable

**Comment by 0xced** [comment]
Thanks for merging and releasing version 0.47.0! 🎉

**Pull request merged**